src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+
+    private double radius;
+    private String color;
+
+    public Circle(double radius, String color) {
+        this.radius = radius;
+        this.color = color;
+    }
+
+    public double uniqueMethod() {
```

Comment:
from the method name i can't understend what it does. Method name should represent the logic that is executed inside

Reply:
done - changed to more specific description